backend/src/main/java/com/woowacourse/thankoo/coupon/application/dto/CouponMessage.java

```diff
@@ -5,24 +5,36 @@
 import com.woowacourse.thankoo.member.domain.Name;
 import java.text.MessageFormat;
 import java.util.List;
+import java.util.Map;
 import lombok.AccessLevel;
 import lombok.NoArgsConstructor;
 
 @NoArgsConstructor(access = AccessLevel.PRIVATE)
 public class CouponMessage {
 
-    public static final String SENDER = "보내는 이 : {0}";
-    public static final String TITLE = "제목 : {0}";
-    public static final String TYPE = "쿠폰 종류 : {0}";
-    public static final String PRETEXT = "\uD83D\uDC8C 커피 쿠폰이 도착했어요.";
+    private static final String SENDER = "보내는 이 : {0}";
+    private static final String TITLE = "제목 : {0}";
+    private static final String TYPE = "쿠폰 종류 : {0}";
+    private static final String COFFEE_PRETEXT = "\uD83D\uDC8C 커피 쿠폰이 도착했어요.";
```

Comment:
이모티콘들이 어떤 이모티콘인지 잘 모르겠는어, 얘네를 별도로 분리하는 건 어때?
```suggestion
    private static final String HEART_LETTER_EMOJI = "\uD83D\uDC8C";
    private static final String COFFEE_PRETEXT = HEART_LETTER_EMOJI + " 커피 쿠폰이 도착했어요.";
```